### PR TITLE
APP-4800 Adding support dark mode

### PR DESCRIPTION
**Pull request by gabrielsetradantas**
## Description

Included a way to do the validation using visual testing in 3 modes:
- Normal mode
- Dark mode
- Dark mode + Condensed Theme

### Without Dark and Dark_condensed mode images
image

### After Images Added

image

### Total Duration Comparsion
**Actual** 
image

**Previous**
image

### CODE REVIEW

**Comment by gabrielsetradantas** [comment]
ok to test

File: .storybook/preview.js [hunk 1]
```diff
@@ -32,4 +35,22 @@ const init = () => {
       document.body.classList.remove(darkClass);
     }
   });
+  channel.on('CONDENSED_MODE', (isCondensed) => {
```
Comment (symphony-adnane) [inline .storybook/preview.js:38]: I don't think we need this one

File: .storybook/preview.js [hunk 2]
```diff
@@ -32,4 +35,22 @@ const init = () => {
...
+    } else {
+      document.body.classList.remove(condensedClass);
+    }
+  });
+  channel.on(Events.GLOBALS_UPDATED,(context) => {
+    const isDark = context.globals.dark;
+    const isCondensed = context.globals.condensed;
+    if (isDark) {
+      document.body.classList.add(darkClass);
+      document.body.style.background = '#1A1C1D'
```
Comment (symphony-adnane) [inline .storybook/preview.js:50]: I think this color code is used above can we put it in a constant ?

File: .storybook/preview.js [hunk 3]
```diff
@@ -32,4 +35,22 @@ const init = () => {
...
     }
   });
+  channel.on('CONDENSED_MODE', (isCondensed) => {
+    if (isCondensed) {
+      document.body.classList.add(condensedClass);
+    } else {
+      document.body.classList.remove(condensedClass);
+    }
+  });
+  channel.on(Events.GLOBALS_UPDATED,(context) => {
```
Comment (symphony-jean-michael) [inline .storybook/preview.js:45]: 👍

**Review by symphony-jean-michael — APPROVED** [review]
LGTM

**Comment by gabrielsetradantas** [comment]
@symphony-adnane could you review it again after your requests?

File: .creevey/config_darkmode.js [hunk 4]
```diff
@@ -0,0 +1,29 @@
+const path = require('path');
+
+module.exports = {
```
Comment (symphony-adnane) [inline .creevey/config_darkmode.js:3]: we need to refactor these files and change only what's needed by extending a root one 
you can create a task to improve this ;)

**Review by symphony-adnane — APPROVED** [review]

**Pull request merged**